config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ matter_add_gn_arg_bool  ("chip_detail_logging"                    CONFIG_MATTER_
 matter_add_gn_arg_bool  ("chip_automation_logging"                FALSE)
 matter_add_gn_arg_bool  ("chip_malloc_sys_heap"                   CONFIG_CHIP_MALLOC_SYS_HEAP)
 matter_add_gn_arg_bool  ("chip_enable_wifi"                       CONFIG_WIFI_NRF700X)
+matter_add_gn_arg_bool  ("chip_system_config_provide_statistics"  CONFIG_CHIP_STATISTICS)
```

Comment:
Similarly as in the comment above, I would consider including shell in name and maybe making it shorter, like `chip_system_statistics_shell`.